crawl-ref/source/mon-info.cc

```diff
@@ -1399,6 +1405,11 @@ vector<string> monster_info::attributes() const
 
     if (is(MB_POISONED))
         v.emplace_back("poisoned");
+    else if (is(MB_MORE_POISONED))
+        v.emplace_back("very poisoned");
+    else if (is(MB_MAX_POISONED))
+        v.emplace_back("extremely poisoned");
```

Comment:
Grammar is a little odd here, but I can't think of a good alternative :/

Reply:
I'm not sure; if we expand this to show all 4 levels, perhaps it should be poisoned, very poisoned, extremely poisoned, completely poisoned? 

'completely poisoned' sounds weird to my ears though.